STYLE_API_TEMPLATE.md

```diff
@@ -135,3 +135,58 @@ PUT /sample-index/_mapping
   }
 }
 ```
+
+### Response body
+
+Provide a description of the response fields.
```

Comment:
I would provide a description of the response fields after the sample response. In my view, it has to be the opposite of request. So, the order would be:
Request body fields
Sample request
Sample response
Response body fields.
This is because to build a request you need to know the body fields beforehand. Then, you get a response and you view the description of its fields.

Reply:
Restructured this based on the outline above.